Scalar.Common/Maintenance/ConfigStep.cs

```diff
@@ -240,7 +238,8 @@ private void ConfigureWatchmanIntegration()
                 if (string.IsNullOrEmpty(hooksDir))
                 {
                     this.Context.Tracer.RelatedError("Could not find hook templates directory. Skipping watchman integration.");
-                    return;
+                    error = null;
+                    return true;
```

Comment:
I understand that a warning could `return true` but why would an error?  Does this need to be changed to a `RelatedWarning`?

Reply:
Because this one should return false. Thanks!